packages/components/src/Field/Field.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
+import { Label, ValidationMessage } from '@marigold/components';
+import { Box, system } from '@marigold/system';
+import { Exclamation, Required } from '@marigold/icons';
+
+type FieldProps = {
+  htmlFor?: string;
+  label?: string;
+  error?: string;
+};
+
+export const Field = system<FieldProps, 'input'>(
+  ({
+    variant = 'field',
+    type = 'text',
+    label,
+    htmlFor = 'id',
+    error,
+    ...props
+  }) => {
+    return label ? (
+      <div>
+        <Label htmlFor={htmlFor} variant="fieldLabel">
+          {label}
+          {error ? (
+            <Required
+              size={16}
+              fill="#dd4142"
+              css={{
+                ml: 1,
+              }}
+            />
+          ) : (
+            ''
+          )}
+        </Label>
+        <Box
+          as="input"
+          type={type}
+          themeSection="form"
+          variant={variant}
+          {...props}
+          css={
+            error
+              ? {
+                  border: '1px solid #dd4142',
```

Comment:
From theme? Theme-ui support borders, we could create a general "borders.error" or something :)

Reply:
ok sure. I had not a good idea for this.

Reply:
@sebald can you help me with this? I don't know how to use this additionally to the normal variant.